src/RTT_Format_Reader/CellDefs.cc

```diff
@@ -50,16 +50,16 @@ void CellDefs::readDefs(ifstream & meshfile)
 
     for (size_t i = 0; i < static_cast<size_t>(dims.get_ncell_defs()); ++i)
     {
-	meshfile >> cellDefNum >> dummyString;
```

Comment:
These are the type of style-based diffs we want eliminate from the code review process by enforcing a standard. No change is needed. This will get cleaned up once #58 is complete.